app/js/controllers/stopsCtrl.js

```diff
@@ -96,10 +96,7 @@
                     var platform = ionic.Platform.platform();
                     $scope.platform = platform;
                     var atMobileSplashData = platform === 'android' ? appConfig.atMobileSplash.android : appConfig.atMobileSplash.ios;
-                    $scope.atMobileSplash = angular.extend(atMobileSplashData, {
-                        isDecommisioned: atMobileSplashData.decommissioningDate && atMobileSplashData.decommissioningDate.isBefore(moment()),
-                        decommissioningDate: atMobileSplashData.decommissioningDate && atMobileSplashData.decommissioningDate.format('D MMMM YYYY')
-                    });
+                    $scope.atMobileSplash = atMobileSplashData;
```

Comment:
Why using a temporary variable here? Can't we just assign the content of atMobileSplashData to $scope.atMobileSplash?